exercises/concept/squeaky-clean/src/test/java/SqueakyCleanTest.java

```diff
@@ -38,6 +38,10 @@ public void ctrl() {
     public void string_with_no_letters() {
         assertThat(SqueakyClean.clean("\uD83D\uDE00\uD83D\uDE00\uD83D\uDE00")).isEmpty();
     }
+    @Test
+    public void remove_no_letters() {
```

Comment:
```suggestion

    @Test
    public void keep_only_letters() {
```

Maybe this is a more appropriate name for the test?